src/server/tcp.rs

```diff
@@ -219,6 +219,12 @@ mod addr_stream {
         pub fn remote_addr(&self) -> SocketAddr {
             self.remote_addr
         }
+
+        /// Consumes the AddrStream and return the underlying IO object
```

Comment:
Copy editing nit: ”Consumes the AddrStream and *returns* the underlying IO object“